test/interpret.jl

```diff
@@ -645,7 +645,14 @@ module VecTest
     end
     f() = 1.0 * one(Tensor{2,3})
 end
-let a = (VecElement{Float64}(1.0), VecElement{Float64}(2.0))
+let
+    # NOTE we need to make sure this code block is compiled, since vecadd is generated function,
+    # but currently `@interpret` doesn't handle a call to generated functions very well
+    @static if isdefined(Base, :Experimental) &&
+               isdefined(Base.Experimental, Symbol("@force_compile"))
+        Base.Experimental.@force_compile
+    end
+    a = (VecElement{Float64}(1.0), VecElement{Float64}(2.0))
```

Comment:
You sure you need this? It didn't fail for me on nightly.

Reply:
Yeah, I couldn't reproduce the error within REPL, but I also confirmed `Pkg.test` will hit [this line](https://github.com/JuliaDebug/JuliaInterpreter.jl/blob/398721d10be2edf40883a11b98e210ccda5393e3/src/construct.jl#L271) consistently without this change.
I think the root problem here is that `@interpret` doesn't configure `enter_generated` when calling `enter_call_expr`.

Reply:
I'm OK with this if it's necessary.